sources/packages/backend/apps/db-migrations/src/sql/EducationPrograms/Create-education-programs-history-table.sql

```diff
@@ -0,0 +1,187 @@
+-- Table to keep historical data.
+-- Columns history_timestamp and history_operation should be default columns for all the history tables.
+-- Columns other than the above-mentioned should reflect the columns from the original table without constraints and relationships.
+-- The order of the columns below follow the exact same order from the existing table.
+CREATE TABLE sims.education_programs_history (
+  history_timestamp TIMESTAMP WITH TIME ZONE NOT NULL,
+  history_operation VARCHAR(20) NOT NULL,
+  id INT,
+  program_name VARCHAR(300),
+  program_description VARCHAR(50),
+  credential_type VARCHAR(50),
+  cip_code VARCHAR(50),
+  noc_code VARCHAR(50),
+  sabc_code VARCHAR(50),
+  regulatory_body VARCHAR(50),
```

Comment:
regulatory_body is VARCHAR(100) 
![image](https://github.com/user-attachments/assets/b84eec2d-a328-4016-8dba-0f18f05d9527)

Reply:
Fixed!